applications/RomApplication/python_scripts/rom_manager.py

```diff
@@ -517,5 +535,15 @@ def _GetDefaulHromTrainingParameters(self):
 
 
 
+    def _StoreSnapshotsMatrix(self, string_numpy_array_name, numpy_array):
+
+        # Define the directory and file path
+        directory = './SnapshotsMatrices'
```

Comment:
Maybe we should do this user definable?

Reply:
We shall be careful not to hyper-parametrise the RomManager. That is one of the reasons I have opted not to pass an extra parameter to the RomManager itself, but pass an optional parameter to its Fit() method. 

Having said that, in an upcoming PR which is being prepared by @SADPR(#10990), multiple ProjectParameters.json will be usable, and therefore the path names will have to be provided. If we can leave it hard-coded (although quite evocative) for now, we will address the issue of allowing users to define their own folder names in that PR.

Reply:
Fair enough.